docs/csharp/language-reference/operators/patterns.md

```diff
@@ -72,6 +76,40 @@ The discard pattern is useful in `switch` expressions. If none of a `switch` exp
 
 The discard pattern cannot be a pattern in an `is` expression or a `switch` statement.
 
+## Relational patterns
+
+Beginning with C# 9.0, you use a *relational pattern* to compare an expression result with a constant, as the following example shows:
+
+:::code language="csharp" source="snippets/patterns/RelationalPatterns.cs" id="BasicExample":::
+
+In a relational pattern, you can use any of the [relational operators](comparison-operators.md) `<`, `>`, `<=`, or `>=`. A constant to compare with can be of an [integral](../builtin-types/integral-numeric-types.md), [floating-point](../builtin-types/floating-point-numeric-types.md), [char](../builtin-types/char.md), or [enum](../builtin-types/enum.md) type.
```

Comment:
You can also use the nullable value types that correspond:

```suggestion
In a relational pattern, you can use any of the [relational operators](comparison-operators.md) `<`, `>`, `<=`, or `>=`. A constant to compare with can be of an [integral](../builtin-types/integral-numeric-types.md), [floating-point](../builtin-types/floating-point-numeric-types.md), [char](../builtin-types/char.md), or [enum](../builtin-types/enum.md) type, or the corresponding nullable value type.
```

Reply:
Hm, constant at the right-hand side cannot be null. But can it be of a nullable type? This sentence is about the type of constant, not expression. I'll check that, thank you.

Reply:
You're right. I misread this originally. Consider this update instead:

```suggestion
In a relational pattern, you can use any of the [relational operators](comparison-operators.md) `<`, `>`, `<=`, or `>=`. A constant to compare with can be of an [integral](../builtin-types/integral-numeric-types.md), [floating-point](../builtin-types/floating-point-numeric-types.md), [char](../builtin-types/char.md), or [enum](../builtin-types/enum.md) type. When the left side of the comparison is a nullable value type, the constant on the right side is converted to the corresponding nullable type.
```

Reply:
That sounds too technical. Why to mention nullable value type, but not mention `object`, for example? And actually I  think it's not the constant that is converted, but the expression result is unwrapped to the underlying type.

Reply:
You're right. I didn't like the way it sounded either.

I may be overthinking this, and leaving it exactly as you wrote it is best.